core/src/main/resources/hudson/model/AbstractItem/confirm-rename_tr.properties

```diff
@@ -0,0 +1,3 @@
+DescribeRename={1} {0}sinin Adını Değiştir
```

Comment:
```suggestion
DescribeRename={1} {0} sinin Adını Değiştir
```

Reply:
"si" and "nin" are suffixes in Turkish which needs to be appended to the word without space. The resulting word should be "Proje**sinin**" not "Proje sinin"

Reply:
Thanks for the clarification!